sql/driver/conn.go

```diff
@@ -53,49 +44,31 @@ import (
 // be stateful and is not used concurrently by multiple goroutines; See
 // https://golang.org/pkg/database/sql/driver/#Conn.
 type conn struct {
-	db       *client.DB
-	sender   *httpSender
-	database string
+	db     *client.DB
+	sender *httpSender
 }
 
 func (c *conn) Close() error {
 	return nil
 }
 
-func createCall(sql string, args []driver.Value) sqlwire.Call {
-	// TODO(vivek): Add arguments later
-	return sqlwire.Call{Args: &sqlwire.Request{Sql: sql}, Reply: &sqlwire.Response{}}
-}
-
 func (c *conn) Prepare(query string) (driver.Stmt, error) {
-	s, err := parser.Parse(query)
-	if err != nil {
-		return nil, err
-	}
-	return &stmt{conn: c, stmt: s}, nil
+	return &stmt{conn: c, stmt: query}, nil
 }
 
 func (c *conn) Exec(query string, args []driver.Value) (driver.Result, error) {
-	stmt, err := parser.Parse(query)
-	if err != nil {
-		return nil, err
-	}
-	return c.exec(stmt, args)
+	return c.exec(query, args)
```

Comment:
No reason for the separation between `Exec` and `exec` anymore. I'd just move the `exec` implementation directly here. Ditto for `Query` and `query`.